src/files/BrsFile.ts

```diff
@@ -222,8 +223,8 @@ export class BrsFile {
         }
 
         let statements = [
-            ...this.parser.libraryStatements,
-            ...this.parser.importStatements
+            ...this.parser.references?.libraryStatements ?? [],
```

Comment:
Optional needed?

Reply:
Nope. It's guaranteed to exist, so no need for optional. Thanks.